### PR TITLE
Add stop channel to controller&cloudhub

### PR DESCRIPTION
**Pull request by fisherxu**
**What type of PR is this?**

> /kind bug

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:
/cc @rohitsardesai83 @sids-b

### CODE REVIEW

File: cloud/pkg/controller/controller.go [hunk 1]
```diff
@@ -32,30 +34,33 @@ func (ctl *Controller) Group() string {
...
+		log.LOGGER.Warnf("new downstream controller failed with error: %s", err)
+		os.Exit(1)
 	}
+	downstream.Start()
+
+	<-ctl.stopChan
+	log.LOGGER.Warnf("stopping upstream controller...")
+	upstream.Stop()
+	log.LOGGER.Warnf("upstream controller stopped")
+	log.LOGGER.Warnf("elected slave, stopping downstream controller...")
```
Comment (rohitsardesai83) [inline cloud/pkg/controller/controller.go]: I don't think we have any leader election mechanism , probably we can delete this log.

Comment (fisherxu) [reply cloud/pkg/controller/controller.go]: Agree, Done, ptal. We can add the leader election in the future.

**Pull request merged**